ref(serverless): new options for ignoring sentry errors (#4755)
getsentry/sentry-docs: src/platforms/node/guides/aws-lambda/index.mdx

// `first` and `second` errors are captured
```

## Ignore Sentry errors
_(New in version 6.17.10)_

> By default, Sentry fails lambda invocation if it unable to send reports to the server.

Reviewer: ```suggestion
By default, Sentry fails lambda invocation if it is unable to send events to Sentry.
```

In this case, even if all reported errors were handled and your handler executed successfully, Sentry will fail whole lambda invocation.

The `ignoreSentryErrors` (default: `false`) option ignores all the errors raised by Sentry

```javascript {tabTitle:ignoreSentryErrors}

> exports.handler = Sentry.AWSLambda.wrapHandler(yourHandler, { ignoreSentryErrors: true });
> // it will ignore any errors raised by sentry sdk on attempt to send reports to the server

Reviewer: ```suggestion
exports.handler = Sentry.AWSLambda.wrapHandler(yourHandler, {
  // Ignore any errors raised by the Sentry SDK on attempts to send events to Sentry
  ignoreSentryErrors: true,
});
```

// `first` and `second` errors are captured
```

## Ignore Sentry Errors

> _(New in version 6.17.10)_

Reviewer: Released in https://github.com/getsentry/sentry-javascript/releases/tag/6.18.0

```suggestion
_(New in version 6.18.0)_
```

By default, Sentry fails lambda invocation if it is unable to send events to Sentry.

> In this case, even if all reported errors were handled and your handler executed successfully, Sentry will fail whole lambda invocation.

Reviewer 2: ```suggestion
In this case, even if all reported errors were handled and your handler executed successfully, Sentry will fail the whole Lambda invocation.
```
